pkgs/applications/video/shotcut/default.nix

```diff
@@ -62,18 +57,18 @@ mkDerivation rec {
     sed "s_/usr/bin/nice_''${NICE}_" -i src/jobs/meltjob.cpp src/jobs/ffmpegjob.cpp
   '';
 
+  patches = [
+    # remove when updating to 23.11
+    ./dont-shadow-final-properties.patch
+  ];
+
   qtWrapperArgs = [
     "--prefix FREI0R_PATH : ${frei0r}/lib/frei0r-1"
     "--prefix LADSPA_PATH : ${ladspaPlugins}/lib/ladspa"
```

Comment:
Was this actually tested? all of the previous PRs have the missing plugin regression mentioned in https://github.com/NixOS/nixpkgs/pull/201905#discussion_r1320980705 which doesn't seem to have been addressed here.

Reply:
thanks for pointing this out, I didn't notice it while testing. frei0r video effects all seem to be present, but ladspa audio filters are indeed missing.

i tried switching LADSPA_PATH to `--set` (from `--prefix`) since other video editing software in nixpkgs seems to use that instead, but sadly without results

Reply:
I think the reason is `melt` not recognizing the filters: they do not show up in `melt -query filters`, and there should be for example `melt -query filter=ladspa.1890`, which is not. I'm trying to compare with older versions where it  worked but I did not succeed yet (maybe because of glibc/libmltjackrack/libjack runtime linking insanities).

Reply:
It was apparently broken by 5f702e92207a43e189a180950f6dc6f8c4d22cfd.
`jackrack/filter_ladspa.yml` is just missing from mlt.

The LADSPA filters reappear and work if mlt is overridden with
`shotcut = qt6Packages.callPackage ../applications/video/shotcut { mlt = mlt.override { enableQt = true; }; };`
but this pulls Qt5 back in, which is not what we want, I guess.
So the fix should be done in mlt package, replacing qt5 with qt6 there and enabling it for shotcut.